project-base/storefront/gtm/handlers/onGtmTransportChangeEventHandler.ts

```diff
@@ -3,14 +3,18 @@ import { TypeTransportWithAvailablePaymentsAndStoresFragment } from 'graphql/req
 import { getGtmTransportChangeEvent } from 'gtm/factories/getGtmTransportChangeEvent';
 import { GtmCartInfoType } from 'gtm/types/objects';
 import { gtmSafePushEvent } from 'gtm/utils/gtmSafePushEvent';
+import { CurrentCustomerType } from 'types/customer';
 
 export const onGtmTransportChangeEventHandler = (
     gtmCartInfo: GtmCartInfoType | undefined | null,
     updatedTransport: TypeTransportWithAvailablePaymentsAndStoresFragment | null,
     updatedPickupPlace: TypeListedStoreFragment | null,
     paymentName: string | undefined,
+    user: CurrentCustomerType | null | undefined,
 ): void => {
     if (gtmCartInfo && updatedTransport !== null) {
-        gtmSafePushEvent(getGtmTransportChangeEvent(gtmCartInfo, updatedTransport, updatedPickupPlace, paymentName));
+        gtmSafePushEvent(
+            getGtmTransportChangeEvent(gtmCartInfo, updatedTransport, updatedPickupPlace, paymentName, user),
+        );
```

Comment:
Why was this event changed? The AC do not mention it.

Reply:
Wrong event.